packages/kbn-apm-synthtrace/src/lib/apm/serverless_function.ts

```diff
@@ -16,6 +16,7 @@ export class ServerlessFunction extends Entity<ApmFields> {
       ...this.fields,
       ['service.node.name']: instanceName,
       'host.name': instanceName,
+      'host.architecture': 'arm',
```

Comment:
in order to generate more realistic with synthrace wouldn't be better to pass the architecture as variable? 

so we could have both `  'arm'  'x86_64' `